src/pages/dataset-view/dataset-view.js

```diff
@@ -21,6 +21,8 @@ export default class extends Component {
 
 
     componentDidMount() {
+        this.props.setQuery('')
```

Comment:
(Tony) Is this the default state for the query string in redux?  Maybe we should make this function more generic (something like `clearQuery()`) and let the reducer put it back to the empty string.